src/configuration/configuration.ts

```diff
@@ -129,6 +129,17 @@ class Configuration implements IConfiguration {
       this.wrapKeys[wrapKey] = true;
     }
 
+    // set keymodel options
+    const km = this.keymodel.split(',');
+    this.keymodelStartsSelection = km.includes('startsel');
+    this.keymodelStopsSelection = km.includes('stopsel');
+
+    // set selectmode options
+    const sm = this.selectmode.split(',');
```

Comment:
I think is better done by defining a getter for `selectmode` which returns a value like this:

```typescript
{
  mouse: boolean,
  key: boolean,
  cmd: boolean,
}
```

(and similar for `keymodel`). Or maybe a separate method (not a real getter) to avoid messing up things like `:set selectmode?`

Reply:
This would definitely look better, but I'm afraid that it won't be as readable and as simple to understand:
```ts
if (configuration.selectModeMouse) {
```
vs
```ts
if (configuration.selectMode().mouse) {
```

I could even change the current implementation adding `By` like this, which could be even more readable:
```ts
if (configuration.selectModeByMouse) {
```
```ts
if (configuration.selectModeByKey) {
```
```ts
if (configuration.selectModeByCmd) {
```